src/mlagility/cli/report.py

```diff
@@ -61,11 +64,11 @@ def _get_estimated_groq_latency(model_folder, cache_folder):
                         f"but got {perf.latency_units}"
                     )
                 )
-            return 1000 * perf.latency
+            return 1000 * perf.latency, gmodel.state.num_chips_used
         else:
-            return "-"
-    except FileNotFoundError:
-        return "-"
+            return "-", "-"
+    except (FileNotFoundError, ImportError):
```

Comment:
Can we add a log warning/error here instead of just returning "-", "-"

Reply:
Can you open an issue if you want that (or other feature requests)? This PR doesn't change any functionality, it just refactors the code.